.github/workflows/deploy.yml

```diff
@@ -1,3 +1,6 @@
+# This GitHub workflow has been disabled and is no longer used. 
+# Website deploys are now done through Cloud Build. 
+# See https://github.com/dart-lang/site-www/pull/5447
```

Comment:
Thanks for adding a comment to explain this. However, can we just delete the file or do you foresee a reason it will be needed again soon?

Reply:
That's a fair point! I was leaving it mostly so that the specification deployment folks can have a reference when moving their stuff over.

Reply:
Ah I see. That's fine then since you disabled it :)

Reply:
It'll be good to remove once all the cloud build migrations are completed, though. I'll file an issue so that we remember.